pkg/apis/machineconfiguration.openshift.io/v1/zz_generated.deepcopy.go

```diff
@@ -241,6 +241,11 @@ func (in *ControllerConfigSpec) DeepCopyInto(out *ControllerConfigSpec) {
 		*out = make([]byte, len(*in))
 		copy(*out, *in)
 	}
+	if in.CloudProviderCAData != nil {
+		in, out := &in.CloudProviderCAData, &out.CloudProviderCAData
+		*out = make([]byte, len(*in))
+		copy(*out, *in)
```

Comment:
The `in` and `out` indirection makes this harder for me to read, although that's obviously subjective.  What about:

```go
if in.CloudProviderCAData != nil {
  out.CloudProviderCAData = make([]byte, len(in.CloudProviderCAData))
  copy(out.CloudProviderCAData, in.CloudProviderCAData)
}
```

?

Reply:
Ah, I guess the `in` and `out` indirection is part of an established code pattern around here; never mind ;).

Reply:
I prefer your code, however 
```
// +build !ignore_autogenerated

// Code generated by deepcopy-gen. DO NOT EDIT.
```